projects/Flarum/default.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+
+{
+  metadata = {
+    summary = "Flarum is a technically advanced, open and extensible discussion platform.";
+    subgrants = [
+      "Flarum"
+    ];
+  };
+
+  nixos.modules.programs = {
+    flarum = {
+      name = "flarum";
+      module = ./programs/Flarum/module.nix;
+      examples.basic = {
+        module = ./programs/Flarum/examples/basic.nix;
+        description = "Discussion community";
+        tests.basic = null;
+      };
+      links = {
+        build = {
+          text = "Installation";
+          url = "https://github.com/flarum/cli#installation";
+        };
+        test = {
+          text = "cli test";
+          url = "https://github.com/flarum/cli/tree/3.x/test";
+        };
+      };
+    };
+  };
+
+  nixos.modules.services = {
+    flarum = {
+      name = "flarum";
+      module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/flarum.nix";
+      examples.basic = {
+        module = ./services/Flarum/examples/basic.nix;
+        description = "Flarum is a delightfully simple discussion platform for your website";
```

This is a description for the example, not the project

```suggestion
        description = "";
```